### PR TITLE
Changes in buttons

### PR DESCRIPTION
**Pull request by dontwike**
Hey,
I have added the icons in the NewPost button and Logout button and also added some CSS. please review the code

### CODE REVIEW

**Comment by narayan954** [comment]
Hi can you resolve the conflicts so I can have a look?

File: src/custom.css [hunk 1]
```diff
@@ -0,0 +1,7 @@
+.btn:hover {
```
Comment (narayan954) [inline src/custom.css:1]: why is there need to create a separate file when index.css exists

**Review by narayan954 — CHANGES_REQUESTED** [review]
fix the conflicts so I can take a better view

File: src/custom.css (hunk 1 above)
Comment (dontwike) [reply src/custom.css:1]: ok i will set it in the index.css file

**Comment by dontwike** [comment]
issue number is #288, in which i added icon in logout button

**Comment by narayan954** [comment]
> issue number is #288, in which i added icon in logout button

but you weren't assigned. You should fix the issue you're assigned and also that issue was outdated. I'll looking into this but update that issue and get yourself assigned there asap

File: src/App.jsx [hunk 2]
```diff
@@ -23,11 +23,17 @@ import Post from "./components/Post";
 import PostView from "./pages/PostView";
 import Profile from "./pages/Profile";
 import ShareModal from "./components/ShareModal";
+
+import { HiOutlineLogout } from "react-icons/hi";
+import { RiImageAddFill } from "react-icons/ri";
+import './index.css'
```
Comment (narayan954) [inline src/App.jsx:29]: it's not needed as the css is loaded by index.js itself

File: src/App.jsx [hunk 3]
```diff
@@ -207,20 +215,42 @@ function App() {
...
           style={{
             cursor: "pointer",
+            marginLeft: "4%"
           }}
         />
 
         {user ? (
           <>
+
+            <Button onClick={() => setOpenNewUpload(true)} sx={buttonStyle}>
```
Comment (narayan954) [inline src/App.jsx:225]: why's there duplicate code/button? I think while resolving conflicts you mistakenly added both parts of conflicted code

File: src/App.jsx [hunk 4]
```diff
@@ -207,20 +215,42 @@ function App() {
...
               variant="contained"
               sx={buttonStyle}
             >
               <AddCircleOutlineIcon style={{ marginRight: "4" }} />
+
               New Post
+              </div>
             </Button>
+
+            <Button
```
Comment (narayan954) [inline src/App.jsx:241]: another duplicate element

File: src/components/Post.jsx [hunk 5]
```diff
@@ -295,6 +295,55 @@ function Post(prop) {
               </DialogActions>
             </Dialog>
           </div>
+
+
+
+          <span style={{ marginLeft: "", fontWeight: "bold" }}>
```
Comment (narayan954) [inline src/components/Post.jsx:301]: again duplicate copy (line 453 perhaps)

**Review by narayan954 — CHANGES_REQUESTED** [review]
Please test the code. Perhaps while resolving conflict you added the conflicting code as well. Please remove the conflicting code and test/verify the application

**Comment by narayan954** [comment]
@dontwike I'd have to close this as unattended.

**Pull request closed without merging**